makefile

```diff
@@ -56,7 +56,7 @@ $(buildDir)/run-linter:cmd/run-linter/run-linter.go $(buildDir)/.lintSetup $(bui
 
 # benchmark setup targets
 $(buildDir)/run-benchmarks:cmd/run-benchmarks/run_benchmarks.go $(buildDir)
-	$(goEnv) $(gobin) build -o $@ $<
+	$(goEnv) $(gobin) build -ldflags "-w" -o $@ $<
```

Comment:
Why the ldflags?

Reply:
it doesn't run on catalina without them

Reply:
What do you mean "doesn't run"? I just tried running it without ldflags and it started fine.

Reply:
`dyld: malformed mach-o image: segment __DWARF has vmsize < filesize`

Reply:
i get this error without the ldflags

Reply:
I can't reproduce. Can you remove it and we can look at this later?